backend/src/views.py

```diff
@@ -229,4 +233,52 @@ def process_daybydays(request):
     # Calculate daybydays
     daybydays = generate_daybydays(ExecutionContext(parameters, rules))
 
-    return Response({ "daybydays": daybydays })
\ No newline at end of file
+    return Response({ "daybydays": daybydays })
+
+
+@api_view(['GET'])
+def export_transactions(request):
+    userid = _get_userid(request)
+    if not userid:
+        return Response({ "message": "`userid` query param is required" }, status=status.HTTP_400_BAD_REQUEST)
+
+    # Pull out parameters
+    parameters = None
+    try:
+        parameters = make_execution_parameters(request)
+    except ValueError as e:
+        return Response({ "error": str(e) }, status=status.HTTP_400_BAD_REQUEST)
+    except AssertionError as e:
+        return Response({ "error": str(e) }, status=status.HTTP_400_BAD_REQUEST)
+    except Exception as e:
+        print(e)
+
+    rules = None
+    try:
+        rules = get_rules_from_database(userid)
+    except Exception as e:
+        logging.error(f"Error while getting rules from database", e)
+        return Response({ "message": "Internal Server Error" }, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+    transactions = get_transactions_up_to(ExecutionContext(parameters, rules))
+    results = list(map(lambda i: i.serialize(), transactions))
+
+    response = HttpResponse(content_type='text/csv')
+    fileName = "MoneyWiseTransactions." + parameters.start.strftime('%m%d%Y') + "." + parameters.end.strftime('%m%d%Y') + ".csv" 
```

Comment:
```suggestion
    fileName = "MoneyWiseTransactions." + parameters.start.strftime('%Y-%-m-%-d') + "." + parameters.end.strftime('%Y-%-m-%-d') + ".csv" 
```
"2020-12-06" format means files will sort meaningfully, can we give this a try? (The %-d, _should_ be adding a leading 0, if applicable, but might be platform-specific: https://strftime.org/)

Reply:
Ow man, looks like it doesn't do the leading 0's. OK, still, thanks for making the change.